analyzer/src/main/java/org/jboss/gm/analyzer/alignment/LockfileIO.java

```diff
@@ -0,0 +1,73 @@
+package org.jboss.gm.analyzer.alignment;
```

Comment:
I wonder if this should be in common/ .... / io for consistency ?

Reply:
I was thinking of it, but since it won't be used in the other module/plugin, I opted to keep it simple for the time being

Reply:
Yeah see what you're saying. I think I would still prefer it there for consistency (as that packaging is becoming a bit of a 'dumping ground' ... either that or we create a new io subpackage ?

Reply:
OK, I'll add it to a new io subpackage :+1:

Reply:
But we can of course move it to common if you prefer, up to you!